File: app/src/main/java/br/com/victorwads/parkingshare/ParkingShareApplication.java
package br.com.victorwads.parkingshare;

import android.app.Application;

import com.google.firebase.firestore.FirebaseFirestore;

public class ParkingShareApplication extends Application {

    @Override
    public void onCreate() {
        if (BuildConfig.DEBUG) FirebaseFirestore.getInstance()
                .useEmulator("192.168.31.75", 8090);

        super.onCreate();
    }
}

File: app/src/main/java/br/com/victorwads/parkingshare/ParkingShareApplication.kt
package br.com.victorwads.parkingshare

import android.app.Application
import com.google.firebase.firestore.FirebaseFirestore

class ParkingShareApplication : Application() {
    override fun onCreate() {
        if (isDebug) FirebaseFirestore.getInstance()
            .useEmulator("192.168.31.75", 8090)
        super.onCreate()
    }
}
